ui/tests/acceptance/settings/mount-secret-backend-test.js

```diff
@@ -298,4 +300,61 @@ module('Acceptance | settings/mount-secret-backend', function (hooks) {
       `${v1} navigates to list route`
     );
   });
+
+  module('Community', function (hooks) {
+    hooks.beforeEach(function () {
+      this.version = this.owner.lookup('service:version');
+      this.version.type = 'community';
+      this.store = this.owner.lookup('service:store');
+    });
+
+    test('identity_token_key form field does not show even when an oidc key exists', async function (assert) {
+      // create an oidc/key
+      await runCmd(`write identity/oidc/key/some-key allowed_client_ids="*"`);
+      await page.visit();
```

Comment:
Can you update these with the new test selector patterns? I think in new tests we should avoid using `ember-cli-page-object` helpers so there are fewer test updates when we migrate away from them completely

Reply:
I pinged in the slack channel about this and opted to have a nice-to-have ticket for replacing all test selectors after the fact. It was a chicken or egg, but I opted to keeping the scope super tight on these prs.